### PR TITLE
:bug: Prefer originalTitle for analytics, if set.

### PR DESCRIPTION
**Pull request by brandondiamond**
Prefers "document.originalTitle" over "document.title" for analytics, if set. This addresses a regression with analytics.

### CODE REVIEW

File: src/service/url-replacements-impl.js [hunk 1]
```diff
@@ -195,7 +195,8 @@ export class GlobalVariableSource extends VariableSource {
 
     // Returns the title of this AMP document.
     this.set('TITLE', () => {
-      return this.ampdoc.win.document.title;
+      return this.ampdoc.win.document.originalTitle ||
```
Comment (cramforce) [inline src/service/url-replacements-impl.js]: Nit: `['originalTitle']`

File: src/service/url-replacements-impl.js [hunk 2]
```diff
@@ -195,7 +195,8 @@ export class GlobalVariableSource extends VariableSource {
...
+          this.ampdoc.win.document.title;
```
Comment (cramforce) [inline src/service/url-replacements-impl.js]: Please add a comment like

The environment may override the title and set `originalTitle`. Prefer that if available.

**Comment by lannka** [comment]
I suppose this originalTitle was written externally?

**Comment by brandondiamond** [comment]
Yup! It is now added in the viewer / AMP integration JS.

**Comment by jridgewell** [comment]
> Yup! It is now added in the viewer / AMP integration JS.

Huh? Where?

**Comment by cramforce** [comment]
@jridgewell Added by a non-open-source viewer implementation in the Google AMP Cache that is used in Chrome Custom Tab (as opposed to the default viewer that is open source).

**Pull request merged**